### DISCUSSION TITLE
Subpath configuring for backend and UI

### DISCUSSION THREAD

**Discussion by akshayjain3450** [rilldata/rill#4290] [General]
Hi.

I am looking to host the rill local I have on localhost:9009/personal. I want to know what all changes I will require for rill to work without any challenges and the minimal way to achieve this.

Thank You

**Comment by AndrewRTsao** [comment]
Hey @akshayjain3450 unfortunately Rill needs to be hosted at the domain root and doesn't support using a subpath at the moment. Besides serving of the frontend, this would require additional backend changes in terms of dynamic path prefixes and API requests. You can find more details in this previous thread here: https://github.com/rilldata/rill/issues/3720

If this is something you're interested in, would love to hear more about your specific use case and we can try getting this captured as a potential feature request in our backlog for further review by our product team.

**Reply by akshayjain3450** [reply]
@AndrewRTsao I will work over this based on my use-case and try to generalise sub-path and create a PR for the same with time and submit for your product team to review. Thank you for your answer.

**Reply by AndrewRTsao** [reply]
Sounds good - thank you!
